packages/api-docs/src/tableRowJson.ts

```diff
@@ -2,7 +2,7 @@ import {
   ApiDeclaredItem,
   ExcerptToken,
   IExcerptTokenRange,
-  ApiItemKind,
+  /* ApiItemKind, */
```

Comment:
if we are going to comment them out, at least use the same comment syntax :D
```suggestion
  // ApiItemKind,
```

Reply:
good catch, that should be removed completely. ty!